migration/migrating_3_4/understanding-migration-toolkit.adoc

```diff
@@ -1,5 +1,5 @@
-[id="migrating-application-workloads-3-to-4"]
-= Migrating application workloads from {product-title} 3.7 to {product-version}
+[id="understanding-migration-toolkit_{context}"]
+= Understanding the migration toolkit
```

Comment:
Just an observation that I don't see any other references to this set of tools as the "migration toolkit", so just wondering if we should be generically saying that. Also, this reminds me of the "Red Hat Application Migration Toolkit": https://access.redhat.com/documentation/en-us/red_hat_application_migration_toolkit/

Reply:
It's a gradual move toward the rebranding. I have another branch in progress. CAM is going to become the "Migration Toolkit for Containers". RHAMT will be MTA and CNV (OpenShift virtualization)/IMS will be MTV.

In this context, the term 'toolkit' is fairly generic, so it seemed OK to me. Official rebranding date is Aug. 10, not far off.

Reply:
Okay cool, just wanted to make sure RHAMT was being considered in this. Glad to hear that there is a plan for it all!